monkey/tests/unit_tests/infection_monkey/post_breach/actions/test_users_custom_pba.py

```diff
@@ -43,13 +44,18 @@ def fake_custom_pba_linux_options():
         "linux_filename": CUSTOM_LINUX_FILENAME,
         "windows_command": "",
         "windows_filename": "",
-        # Current server is used for attack telemetry
-        "current_server": CUSTOM_SERVER,
     }
 
 
-def test_command_linux_custom_file_and_cmd(fake_custom_pba_linux_options, set_os_linux):
-    pba = CustomPBA(MagicMock())
+@pytest.fixture
+def control_client():
+    return ControlClient(CUSTOM_SERVER)
+
+
+def test_command_linux_custom_file_and_cmd(
+    fake_custom_pba_linux_options, set_os_linux, control_client
+):
+    pba = CustomPBA(MagicMock(), control_client)
```

Comment:
Pass `MagicMock` instead of real `ControlClient` to avoid unintended side-effects.